examples/third_party/python/BUILD.python3.bazel

```diff
@@ -0,0 +1,34 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "configure_make")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "all_srcs",
+    srcs = glob(["**"]),
+)
+
+configure_make(
+    name = "python3",
+    configure_options = [
+        "--with-openssl=$EXT_BUILD_DEPS/openssl",
+        "--with-zlib=$EXT_BUILD_DEPS/zlib",
+    ],
+    env = {"CFLAGS": "-Dredacted='\\\"redacted\\\"'"},
```

Comment:
This workaround shouldn't be needed anymore after #567 was merged - can you try removing this from here and the other locations where this is used?

Reply:
Sadly the issue still occurs, ie _DATE__ and __TIME__ are defined as `redacted` rather than `"redcted"`, therefore this workaround is still required